src/main/java/org/commonjava/indy/service/tracking/change/FoloTrackingListener.java

```diff
@@ -86,6 +86,11 @@ public void handleFileAccessEvent( final FileEvent event )
 
         try
         {
+            String keyString = event.getStoreKey();
+            if (!StringUtils.isNotBlank( keyString )) {
```

Comment:
Why not use "StringUtils.isBlank" instead of this " !StringUtils.isNotBlank"?

Reply:
updated in https://github.com/Commonjava/indy-tracking-service/pull/27